h2h/production-code/week6/get_data.py

```diff
@@ -149,7 +195,7 @@ def str_to_mins(inp):
     sec = int(lst[0])*60 + int(lst[1])
     return sec
 
-def try_to_get(endpoint, parameter, max_retries = 5, timeout = 10):
+def _try_to_get(endpoint, parameter, max_retries = 5, timeout = 10):
     """ Quick method to try to connect tp the NBA API endpoint. The NBA API is
     notoriously annoying to connect to, so this method tries to circumvent that.
     Has different uses for different endpoints"""
```

Comment:
Putting a basic print statement in the "except" could save you a big headache in the future